coreapi/request.go

```diff
@@ -244,3 +248,14 @@ type Schedule struct {
 	// start_time will be current time and first payment will happen on one interval after current time.
 	StartTime string `json:"start_time,omitempty"`
 }
+
```

Comment:
Also should rename type `Schedule`  to `ScheduleDetails` to maintain naming consistency.

Reply:
done